script.js

```diff
@@ -0,0 +1,150 @@
+const toDoForm = document.querySelector(".toDoForm");
+const toDoInput = toDoForm.querySelector("input");
+const toDoList = document.querySelector(".toDoList");
+const finishedList = document.querySelector(".finishedList");
+const lenTodo = document.getElementById("todo-count");
+const lenFin = document.getElementById("fin-count");
+
+const TODODATA = "todoData"
+const FINDATA = "FinData"
+
+let toDoArr = [];
+let finArr = [];
+
+function loadDATA(){
+    const loadedTODO = localStorage.getItem(TODODATA);
+    const loadedFIN = localStorage.getItem(FINDATA);
```

Comment:
이렇다 할 정확한 네이밍 규칙은 존재하지 않지만, 저희 미션에서는 lowerCameCase를 이용하기로 한 만큼 수정해주시면 좋을거 같습니다.
```suggestion
function loadData(){
    const loadedTodo = localStorage.getItem(TODODATA);
    const loadedFin = localStorage.getItem(FINDATA);
```

Reply:
다시 보니까 정말 제멋대로였군요...